GliaWidgets/Glia.swift

```diff
@@ -255,23 +255,27 @@ public class Glia {
             )
         )
         rootCoordinator?.delegate = { [weak self] event in
-            switch event {
-            case .started:
-                self?.onEvent?(.started)
-            case .engagementChanged(let engagementKind):
-                self?.onEvent?(.engagementChanged(engagementKind))
-            case .ended:
-                self?.rootCoordinator = nil
-                self?.onEvent?(.ended)
-            case .minimized:
-                self?.onEvent?(.minimized)
-            case .maximized:
-                self?.onEvent?(.maximized)
-            }
+            self?.handleCoordinatorEvent(event)
         }
         rootCoordinator?.start()
     }
 
+    private func handleCoordinatorEvent(_ event: RootCoordinator.DelegateEvent) {
```

Comment:
But here there is a `handleCoordinatorEvent`. So perhaps the ones at the top should be called different names, or this one should be called `handle`. Personally I prefer the first one, but as long as both are consistent, it's fine.

Reply:
946e38598032d196fd906015b6d7e2a8e44fd974